build.gradle

```diff
@@ -111,6 +116,15 @@ ext {
                 reportThreshold 0.5, BRANCH
             }
         }
+        // Similarly to Zipkin, LightStep doesn't contain much code and the current tracer version doesn't allow for
+        // easy assertions when implemented within a broader tracing framework like Wingtips. We will monitor and continue to make improvements.
+        configure(subprojects.findAll { it.name.contains("wingtips-lightstep") }) {
+            jacocoCoverage {
+                // Enforce minimum code coverage. See https://github.com/palantir/gradle-jacoco-coverage for the full list of options.
+                reportThreshold 0.65, INSTRUCTION
+                reportThreshold 0.35, BRANCH
```

Comment:
This failed in Travis CI. Needs to be a bit lower, at least for the moment.

Reply:
using .4 and .25 respectively